src/main/java/org/hawkular/client/android/backend/BackendClient.java

```diff
@@ -131,8 +132,10 @@ private void configurePipes(URL backendUrl, Persona persona) {
         configurePipe(BackendPipes.Names.FEED_REC_RESOURCES, pipeUrl, pipeModules, Resource.class);
```

Comment:
In hawkular-services 0.0.5 the inventory api has been moved to `/deprecated`.  you may need to change the pipes.
There is a new api under the endpoint that you know, but which is not compatible. So I guess for the moment using `http://<host>:<port>/hawkular/inventory/deprecated` as base is best.